### PR TITLE
TFlite RDT reader

### PR DESCRIPTION
**Pull request by jennya**
Adds tflite RDT to Chills for Android. Still a pretty rough prototype, but basically functional:

https://drive.google.com/a/auderenow.org/file/d/1wp4FJgp2xnt9kYPOPo5GLnBlitVrV6OT/view?usp=sharing

### CODE REVIEW

File: FluStudy_us/src/ui/components/flu/AndroidRDTReader.tsx [hunk 1]
```diff
@@ -0,0 +1,818 @@
...
+    if (!args.testStripFound) {
+      this._lastRDTReaderResult = rdtCapturedArgsToResult(args);
+      return;
+    }
+
+    const { dispatch, navigation, next } = this.props;
+    dispatch(setRDTCaptureTime(true));
+    try {
+      const photoId = await newUID();
+      const hcPhotoId = await newUID();
```
Comment (rightparen) [inline FluStudy_us/src/ui/components/flu/AndroidRDTReader.tsx]: Is high-contrast still relevant?

File: FluStudy_us/src/ui/components/flu/AndroidRDTReader.tsx [hunk 2]
```diff
@@ -0,0 +1,818 @@
...
+      this.setState({
+        fps: ((this._callbackTimestamps.length - 1) / (last - first)) * 1000,
+      });
+      this._callbackTimestamps = this._callbackTimestamps.slice(
+        this._callbackTimestamps.length - 2
+      );
+    }
+  };
+
+  _toggleFlash = () => {
```
Comment (rightparen) [inline FluStudy_us/src/ui/components/flu/AndroidRDTReader.tsx]: Do we still need flash?

File: FluStudy_us/src/ui/components/flu/AndroidRDTReader.tsx (hunk 1 above)
Comment (jennya) [reply FluStudy_us/src/ui/components/flu/AndroidRDTReader.tsx]: We don't do anything to make it high-contrast, but I am using the test area bitmap here which we can display to the user on a later screen to draw attention to the part of the test strip that we care about. I don't think this is necessarily a path we want to go down, but thought it was trivial to implement so left it in for David/Shawna to evaluate.

File: FluStudy_us/src/ui/components/flu/AndroidRDTReader.tsx (hunk 2 above)
Comment (jennya) [reply FluStudy_us/src/ui/components/flu/AndroidRDTReader.tsx]: Maybe not -- will commit as is for now but discuss removing it when we meet with David later today

**Pull request merged**